Enable recording an auditing result message when accepting SCORE

soobokjin opened the pull request:
Add 'warning' parameter on acceptScore() and set governance version to 0.0.6 (#1)
- Supplement README.md about this change

soobokjin committed:
Add 'warning' parameter on acceptScore() (#1)
- Supplement README.md about this change

sink772 commented on governance/governance.py:
```diff
@@ -339,7 +344,7 @@ def setStepPrice(self, stepPrice: int):
 
     @external
-    def acceptScore(self, txHash: bytes):
+    def acceptScore(self, txHash: bytes, warning: str =""):
```
minor thing but it's better to add a space between symbols.  
```suggestion
    def acceptScore(self, txHash: bytes, warning: str = ""):
```

soobokjin committed:
Add calling _set_initial_revision() on on_install (#1)
Remove _set_initial_revision() on _migrate_v0_0_3() as this method is called on _migrate_v0_0_5() (duplicated)
Fix some minor typo in governance

soobokjin committed:
Remove _set_initial_max_step_limits() on _migrate_v0_0_2() as this method is called on _migrate_v0_0_3() (duplicated) (#1)
